problems/loops.js

```diff
@@ -120,30 +194,57 @@ while (i > 3) {
 
 /**
  * // Write your explanation here!
+ *  this leads to an infinite loop because i starts at 5 and will continue to increase, and will always be bigger than 3
+ *
  *
  */
 
- /**
- * Returns an array with all the numbers from all the numbers from 1 to 100.
- * For numbers divisible by 3, add "Fizz" instead of the number into the array.
- * For numbers divisible by 5 (and not 3), add "Buzz" instead of the number into the array.
+/**
+* Returns an array with all the numbers from all the numbers from 1 to 100.
+* For numbers divisible by 3, add "Fizz" instead of the number into the array.
+* For numbers divisible by 5 (and not 3), add "Buzz" instead of the number into the array.
 
- * @returns {Array} [1, 2, "Fizz", 4, "Buzz"... 98, "Fizz", "Buzz"]
-
- */
-function fizzBuzzPart1() {}
+* @returns {Array} [1, 2, "Fizz", 4, "Buzz"... 98, "Fizz", "Buzz"]
 
+*/
+const fizzBuzzPart1 = () => {
+  let arr = [];
+  for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0) {
+      arr.push("Fizz");
+    } else if (i % 5 === 0 && i % 3 !== 0) {
```

Comment:
No need for right side of &&, if it was divisible by 3 it would have entered your first if condition.